plugins/reveal.js

```diff
@@ -30,12 +30,19 @@ class Reveal {
   }
 
   configure() {
-    return this.page.evaluate(fragments => Reveal.configure({
-        controls  : false,
-        progress  : false,
-        fragments : fragments,
-        transition: 'none'
-      }),
+    return this.page.evaluate(fragments => {
+        Reveal.configure({
+          controls  : false,
+          progress  : false,
+          fragments : fragments,
+          transition: 'none'
+        });
+
+        var menuOpenButtons = document.getElementsByClassName('slide-menu-button');
```

Comment:
Would you mind adding a comment that gives a quick context and a reference to https://github.com/denehyg/reveal.js-menu/issues/99?

Reply:
Makes sense, thanks! Done!